### PR TITLE
cc-dashboard pdf and webview links

### PR DESCRIPTION
**Pull request by darrenmason**
regarding task: https://www.pivotaltracker.com/n/projects/1156756/stories/107577054

- [x] pdf link
- [x] webview link
- [x] layout / styling

### CODE REVIEW

File: src/components/cc-dashboard/index.cjsx [hunk 1]
```diff
@@ -90,6 +91,26 @@ DashboardChapter = React.createClass
       {chapter}
       {sections}
     </div>
+
+DashboardBookLinks = React.createClass
+  goToPdf: ->
+    course = CourseStore.get(@props.courseId)
+    window.open(course.book_pdf_url, 'cc-pdf-link')
```
Comment (philschatz) [inline src/components/cc-dashboard/index.cjsx]: Should this get analytics added to it? Is there an `<OpenInNewTab>` component?

/cc @nathanstitt

**Comment by jivewise** [comment]
Going off of the mockups here: http://3lwsnp.axshare.com/#p=teacher_dashboard_-_visual_design

**Comment by jason-holmes** [comment]
I think that’s just an oversight by Fred.  See: http://3lwsnp.axshare.com/#p=teacher_dashboard <http://3lwsnp.axshare.com/#p=teacher_dashboard>.

-------
Jason Holmes, Ph.D.
User Experience Manager
OpenStax | OpenStax.org
(713) 893-1326

**Comment by jason-holmes** [comment]
More specifically, top right of screen above “detailed scores” button.

-------
Jason Holmes, Ph.D.
User Experience Manager
OpenStax | OpenStax.org
(713) 893-1326

File: src/components/cc-dashboard/index.cjsx [hunk 2]
```diff
@@ -90,6 +91,27 @@ DashboardChapter = React.createClass
...
+
+  goToWebview: ->
+    course = CourseStore.get(@props.courseId)
+    window.open("http://cnx.org/contents/#{course.ecosystem_book_uuid}", 'cc-webview-link')
```
Comment (philschatz) [inline src/components/cc-dashboard/index.cjsx]: @jpslav Can BE provide this link (similar to `course.book_pdf_url`)?

Comment (jpslav) [inline src/components/cc-dashboard/index.cjsx]: yes https://github.com/openstax/tutor-server/pull/839 :-)

**Pull request merged**